CONTRIBUTING.md

```diff
@@ -54,16 +54,18 @@ contributions via GitHub pull requests. When contributing code please do the
 following:
 
 * Fork the repository and make your changes in a feature branch.
+* Please include new or update existing PlantUML sequence/architecture diagrams
+  that relate to your PR. ([/docs/diagrams/src/readme.md](details) on how we
+  maintain our diagrams)
 * Please include unit and integration tests for any new features and updates to
-existing tests.
+  existing tests.
 * Please ensure the unit and integration tests run successfully. Refer to the
-Testing section in the [README.md](README.md) on how to run these
-tests.
+  Testing section in the [README.md](README.md) on how to run these tests.
 * Please ensure that lint passes by running `./bin/run_lint`. The command should
-produce no output if there are no lint errors. The linter requires all Python
-dependencies be installed locally (if not it'll throw import-errors). Install
-dependencies locally by running `pip3 install -r requirements.txt` from the
-root project directory.
+  produce no output if there are no lint errors. The linter requires all Python
+  dependencies be installed locally (if not it'll throw import-errors). Install
+  dependencies locally by running `pip3 install -r requirements.txt` from the
```

Comment:
It'll be nice to include a note to be in the virtual environment and install those dependencies in the virtualenv.

Reply:
do we have dev environment setup instructions in `README.md`? if so we might want to just reference the setup instructions in that file instead of duplicating them here.

Reply:
The actual dev env setup is in Sphinx Docs, but I think README.md makes a reference to it. Not 100% sure.

Reply:
yup, just saw that